tendrl_pre.yml

```diff
@@ -0,0 +1,95 @@
+---
+#
+# Tendrl server
+#
+
+- hosts: tendrl-server
+  remote_user: root
+  roles:
+    - role: qe-ssl-cert
+      ssl_cert_name: "etcd"
+      ssl_key_perm: "0644"
+      when: etcd_tls_client_auth|bool == True
+  tasks:
+    - name: Create usmqe user group
+      group:
+        name: usmqe
+        state: present
+
+    - name: Create usmqe user account
+      user:
+        name: usmqe
+        group: usmqe
+        createhome: yes
+        generate_ssh_key: yes
+        comment: "USM QE test user account"
+        state: present
```

Comment:
Hmm, could we reuse some tasks in `qe-server` and `qe-server-user` roles? They are not reusable right now in it's current form, but it's worth the consideration.